frontend/src/libs/Component.js

```diff
@@ -0,0 +1,16 @@
+class Component {
+  constructor($container, initialState) {
+    this.$container = $container;
+    this.state = initialState || {};
+  }
```

Comment:
```suggestion
  constructor($container, initialState = {}) {
    this.$container = $container;
    this.state = initialState;
  }
```

디폴트값으로 명시해주는게 좋을 거 같아 보여😃